### DISCUSSION TITLE
Question Regarding A Feature Availability?

### DISCUSSION THREAD

**Discussion by Ibuprophen** [Serg-Norseman/GEDKeeper#333] [Q&A]
I'm new to this so please be kind to the noob :-))

I'm hoping i can explain this right via text, but I was wondering if there was a way to filter, locate, list, etc... any individuals that are not connected/disconnected within a tree in a gedcom?

I had opened an exported Ancestry GED file and was seeing if there are any disconnected individuals (as explained above)?

I wasn't sure if it was available, but using some type of verbiage i'm not familiar with. If not, this would be something great to implement somehow. :-)) 

Thank you very much for your time and understanding.

~Ibuprophen

**Comment by Serg-Norseman** [comment]
This is a good question :) 👍 

There is such a feature of the application. This is a search for unconnected individuals, families or fragments from several families. It's a module in the menu: "Service \ Tools \ Check connection of families". The module as a whole analyzes and displays the number and composition of fragments in the file, as well as a logarithmic graph of fragmentation.

By right-clicking on any item, you can get quick information about a individual, or go to a item in the main list.

**Comment by Ibuprophen** [comment]
Thank you very much for your response and guidance. I'll give this a go and let you know of the results. 

With so many individuals on my tree, this would definitely be a tedious process manually... LOL! 


~Brian Kerr
